wire-ios/Wire-iOS/Sources/UserInterface/Helpers/UITextView+Autocorrect.swift

```diff
@@ -23,7 +23,6 @@ extension UITextView {
     // Autocorrects the last word, if necessary.
     func autocorrectLastWord() {
         UIView.performWithoutAnimation {
-            resignFirstResponder()
```

Comment:
wait what was the reason before of this? is there a commit pointing to this?
+ which iOS version does this impact?

Reply:
@netbe That "hack" was [for iOS to accept the current auto-correction](https://github.com/wireapp/wire-ios/pull/1104). 

But I tested the current fix with autocorrect on and it works fine. No need to keep the `resignFirstResponder()`